app/models/audit.rb

```diff
@@ -63,4 +64,16 @@ class Audit < ApplicationRecord
   }
 
   validates :activity_type, presence: true
+
+  def validation_request
```

Comment:
For the new audit types I've I've copied the logic we're already using for existing audit types, eg `description_change_validation_request_sent`, where `activity_information` is set to the `sequence` of the request. This means that later on we can derive the specific validation request from `activity_type` and `activity_information`. 

I don't love doing it this way - I think it might be worth having a bigger rethink about audits at some point!